dom/test/browser/src/dom-driver.ts

```diff
@@ -131,8 +131,8 @@ describe('DOM Driver', function() {
     let hasDisposed = false;
     let assertionOngoing = false;
     sources.DOM.select(':root').elements().drop(1).addListener({
-      next: (root: Element) => {
-        const selectEl = root.querySelector('.target') as Element;
+      next: (root: Element[]) => {
+        const selectEl = root[0].querySelector('.target') as Element;
```

Comment:
Instead of introducing the array here, we could take `element` for a spin, and replace `elements()` with `element()`